docs/errors/E207.md

```diff
@@ -0,0 +1,11 @@
+# E207: error escaped code point in Unicode out of range
```

Comment:
Nit: Remove 'error'.

The title should match the error message printed by quick-lint-js.

```suggestion
# E207: code point in Unicode escape sequence must not be greater than U+10FFFF
```